README.md

```diff
@@ -215,6 +215,7 @@ installations requiring long-term consistency.
 | [no-focused-tests](docs/rules/no-focused-tests.md)                           | Disallow focused tests                                              | ![recommended][] | ![suggest][] |
 | [no-hooks](docs/rules/no-hooks.md)                                           | Disallow setup and teardown hooks                                   |                  |              |
 | [no-identical-title](docs/rules/no-identical-title.md)                       | Disallow identical titles                                           | ![recommended][] |              |
+| [no-if](docs/rules/no-if.md)                                                 | Disallow conditional logic                                          | ![deprecated][]  |              |
```

Comment:
@SimenB I guess we forgot to remove this in the new major 😅

Reply:
Yeah, luckily with this change it will be clearly visible that it still exists but is deprecated.

Reply:
I guess - alternatively you could say now its clearly visible that it exists at all even though we don't want people to be using it 🤷

Reply:
My personal preference is to continue documenting deprecated features while clearly indicating they are deprecated. People may still be using the deprecated features and need to find their documentation, and omitting any link to them can cause confusion.